src/app/core/ui/search/search-information/model/search-information.ts

```diff
@@ -0,0 +1,9 @@
+import { Entity } from '../../../../entity/entity';
+import { DatabaseEntity } from '../../../../entity/database-entity.decorator';
+
+
+@DatabaseEntity('SearchInformation')
+export class SearchInformation extends Entity {
```

Comment:
extending from the `Entity` class and using the annotations is important if you want to save instances to the database. If you don't need that, you can use a "normal" class or simple variables without this overhead.